common/service/config/config.go

```diff
@@ -209,7 +209,7 @@ type (
 		// ConnectAttributes is a set of key-value attributes to be sent as part of connect data_source_name url
 		ConnectAttributes map[string]string `yaml:"connectAttributes"`
 		// MaxQPS the max request rate on this datastore
-		MaxQPS int `yaml:"maxQPS"`
+		MaxQPS dynamicconfig.IntPropertyFn `yaml:"-" json:"-"`
```

Comment:
this PR seems to have some overlap with https://github.com/uber/cadence/pull/3062

Reply:
Sync'ed offline about this.